### PR TITLE
Pickups anim

### LINKED ISSUE #235: Improve axe attack
**Description**
Might want to do this together with #207.

**Subtasks**
- [x] Faster attack
- [x] Don't start dealing damage until the axe starts moving forward
- [x] Tweak hitbox?

### LINKED ISSUE #251: Pickups and animations
**Description**
Make held pickups align with and follow the animations.

**Subtasks**
- [x] Hook up item carrying pivot to skeleton
- [x] Hook up axe swing to skeleton

### PR DESCRIPTION
**Pull request by smeas**
Closes #251 
Closes #235 

**Description**  
Linked pickups to player animations.

**List of changes**  
- Made carried items follow the animation
- Linked axe attack to player animation

**Additional context**  
Instrument play animation

### CODE REVIEW

File: Assets/00_Content/Prefabs/Interactables/BeerBarrel.prefab [hunk 1]
```diff
@@ -63,7 +63,8 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: ab5ac90e8d6ee7449933c889df334329, type: 3}
```
Comment (MrJackey) [inline Assets/00_Content/Prefabs/Interactables/BeerBarrel.prefab:63]: I find it a bit odd that the solo carry idle only uses one hand.

**Review by MrJackey — CHANGES_REQUESTED** [review]
I found some quirks, should be taken a look at. Also we don't seem to have an animation for repairing tables, do we want one?

File: Assets/00_Content/Prefabs/Interactables/Instruments/Harp.prefab [hunk 2]
```diff
@@ -159,13 +159,15 @@ MonoBehaviour:
   m_Enabled: 1
   m_EditorHideFlags: 0
   m_Script: {fileID: 11500000, guid: 38d2d1151fed4730952290fdcbd66db8, type: 3}
-  m_Name: 
-  m_EditorClassIdentifier: 
-  itemGrabTransform: {fileID: 0}
+  m_Name:
```
[the remark this answers is not in the document]
Comment (smeas) [reply Assets/00_Content/Prefabs/Interactables/Instruments/Harp.prefab:162]: Will look into it later.

File: Assets/00_Content/Prefabs/Interactables/BeerBarrel.prefab (hunk 1 above)
Comment (smeas) [reply Assets/00_Content/Prefabs/Interactables/BeerBarrel.prefab:63]: Fixed in master.

**Review by MrJackey — APPROVED** [review]
Everything seems to work! :shipit:

**Pull request merged**